resources/smpp/ra/src/main/java/org/restcomm/slee/resource/smpp/SmppSessionsImpl.java

```diff
@@ -495,4 +492,113 @@ public void fireUnknownThrowable(Throwable throwable) {
 
 	}
 
+	public class SenderThread extends Thread {
+
+		private Boolean running = true;
+		private LinkedBlockingQueue<SmppSendingTask> queue = new LinkedBlockingQueue<>();
```

Comment:
we may lazy-init this attribute during run to prevent mem consumption for dicarded SenderThread...

shouldnt we set a certain limit to this queue? ideally should be configurable from env (per esme). This will avoid running out of memory or overloading..